include/arch/armv7/stdint.h

```diff
@@ -146,5 +146,6 @@ typedef uint64_t uintmax_t;
 #define _PRI_16 ""
 #define _PRI_32 ""
 #define _PRI_64 "ll"
+#define _PRI_PTR "ll"
```

Comment:
IMHO this target has 32-bit pointers (see `UINTPTR_MAX` for example) so the correct prefix should be `""`. Applies for other 32-bit arch also.

Reply:
I wasn't sure and wanted someone to verify if it's correct. Thanks, changed.